<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подключиться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="inputName">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chatDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>91</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAllButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
